lib/chemistrykit/cli/cli.rb

```diff
@@ -126,7 +126,8 @@ def rspec_config(config)
         end
 
         ::RSpec.configure do |c|
-          c.capture_log_messages
+          log = Logging.logger['test steps']
+          c.capture_log_messages          
```

Comment:
i'm not entirely sure how it works, but it does. Possibly, Logger.logger['test steps'] is a pointer to a variable that i can add appenders to in the cli script, and then calling the same thing in test_common_library points back to that variable that now has appenders on it.

Like i said, this is just a guess, but this made it work.